tests/foreman/longrun/test_inc_updates.py

```diff
@@ -128,100 +147,66 @@ def setUpClass(cls):
             end_date=date.today().strftime('%Y-%m-%d')
         ).create()
 
-        # Step 10: Publish both the content views and re-read the content views
-
-        # Changing the nailgun timeout time for the rest of the steps as
-        # publish/promote of larger repos take more than 5 minutes
-        entity_mixins.TASK_TIMEOUT = 3600
-        for content_view in (cls.rhel_6_cv, cls.rhel_6_partial_cv):
-            content_view.publish()
-        cls.rhel_6_cv = cls.rhel_6_cv.read()
-        cls.rhel_6_partial_cv = cls.rhel_6_partial_cv.read()
-
-        # Step 11: Promote both content views to 'DEV' and 'QE'
-        for content_view in (cls.rhel_6_cv, cls.rhel_6_partial_cv):
-            assert len(content_view.version) == 1
-            for env in (cls.dev_lce, cls.qe_lce):
-                promote(content_view.version[0], env.id)
-
-        # Step 12: Create host collections
-        for hc_name in ('rhel_6_hc', 'rhel_6_partial_hc'):
-            setattr(
-                cls,
-                hc_name,
-                HostCollection(
-                    organization=cls.org, name=hc_name, max_content_hosts=5
-                ).create()
-            )
-
-        # Step 13: Create activation keys for both content views
-        kwargs = {'organization': cls.org, 'environment': cls.qe_lce.id}
-        rhel_6_ak = ActivationKey(
-            name=u'rhel_6_ak',
-            content_view=cls.rhel_6_cv,
-            host_collection=[cls.rhel_6_hc],
-            **kwargs
-        ).create()
+        # Publish content view and re-read it
+
+        self.rhel_6_partial_cv.publish()
```

Comment:
Originally task timeout here was set to 1 hour, deleting it results in 5 minutes timeout. Maybe it's worth trying, as it passed for you locally, but this may be subject to change in case of failure - for 15 minutes, for example. Let's keep an eye on automation results.